internal/agent/reset.go

```diff
@@ -14,27 +13,32 @@ import (
 	hook "github.com/kairos-io/kairos/v2/internal/agent/hooks"
 	"github.com/kairos-io/kairos/v2/internal/bus"
 	"github.com/kairos-io/kairos/v2/internal/cmd"
+	"github.com/kairos-io/kairos/v2/pkg/action"
 	"github.com/kairos-io/kairos/v2/pkg/config"
 	"github.com/kairos-io/kairos/v2/pkg/config/collector"
+	"github.com/kairos-io/kairos/v2/pkg/elementalConfig"
 
 	"github.com/mudler/go-pluggable"
 	"github.com/pterm/pterm"
 )
 
 func Reset(dir ...string) error {
+	// TODO: Enable args? No args for now so no possibility of reset persistent or overriding the source for the reset
+	// Nor the auto-reboot via cmd?
 	bus.Manager.Initialize()
 
-	options := map[string]string{}
-
 	bus.Manager.Response(sdk.EventBeforeReset, func(p *pluggable.Plugin, r *pluggable.EventResponse) {
-		err := json.Unmarshal([]byte(r.Data), &options)
+		err := json.Unmarshal([]byte(r.Data), &map[string]string{})
```

Comment:
We used to unmarshal to `options` because we needed that variable later. What's the reason of unmarshaling now? Just to print the error if it's not "umarshable"?

Reply:
options was an empty `map[string]string{}` before. This is just removing an extra var. No idea why it was done like this before, I guess because all install,reset,upgrade do the same? In case we add options later?

Reply:
oooh I see, it comes from the `r.Data`.

Where the fuck does that r.Data magically appears from?

Reply:
I have no idea where that comes from or how does it get set to be honest, I'll bring it up on the meeting today.

Also good catch, this was setting reset-persistent by default in no data was coming in, I missed that!